src/test/java/com/oltpbenchmark/catalog/TestHSQLDBCatalog.java

```diff
@@ -0,0 +1,148 @@
+/******************************************************************************
+ *  Copyright 2015 by OLTPBenchmark Project                                   *
+ *                                                                            *
+ *  Licensed under the Apache License, Version 2.0 (the "License");           *
+ *  you may not use this file except in compliance with the License.          *
+ *  You may obtain a copy of the License at                                   *
+ *                                                                            *
+ *    http://www.apache.org/licenses/LICENSE-2.0                              *
+ *                                                                            *
+ *  Unless required by applicable law or agreed to in writing, software       *
+ *  distributed under the License is distributed on an "AS IS" BASIS,         *
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  *
+ *  See the License for the specific language governing permissions and       *
+ *  limitations under the License.                                            *
+ ******************************************************************************/
+
+package com.oltpbenchmark.catalog;
+
+import com.oltpbenchmark.api.MockBenchmark;
+import com.oltpbenchmark.types.DatabaseType;
+import com.oltpbenchmark.util.SQLUtil;
+import junit.framework.TestCase;
+
+import java.io.InputStream;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import static org.junit.Assert.assertNotEquals;
+
+public class TestHSQLDBCatalog extends TestCase {
+
+    private MockBenchmark benchmark;
+    private HSQLDBCatalog catalog;
+
+    @Override
+    protected void setUp() throws Exception {
+        super.setUp();
+
+        this.benchmark = new MockBenchmark();
+        this.catalog = new HSQLDBCatalog(benchmark);
+        assertNotNull(this.catalog);
+    }
+
+    /**
+     * testGetOriginalTableNames
+     */
+    public void testGetOriginalTableNames() {
+        // Make sure that the key and values in this map are not
+        // equal unless we ignore their case
+        Map<String, String> origTableNames = this.catalog.getOriginalTableNames();
+        assertNotNull(origTableNames);
+        assertFalse(origTableNames.isEmpty());
+
+        for (Entry<String, String> e : origTableNames.entrySet()) {
+            assertFalse(e.toString(), e.getKey().equals(e.getValue()));
+            assertTrue(e.toString(), e.getKey().equalsIgnoreCase(e.getValue()));
+        } // FOR
```

Comment:
```suggestion
        }
```

We removed this style recently